include/dsHdmiIn.h

```diff
@@ -793,6 +793,28 @@ dsError_t dsSetEdid2AllmSupport (dsHdmiInPort_t iHdmiPort, bool allmSupport);
  */
 dsError_t dsGetEdid2AllmSupport (dsHdmiInPort_t iHdmiPort, bool *allmSupport);
 
+/**
+* @brief Gets the Maximum HDMI Compatibility Version supported by the given port.
+*
+* For sink devices, this function gets the Maximum HDMI Compatibility Version supported by the given port.
+* For source devices, this function returns dsERR_OPERATION_NOT_SUPPORTED always.
+*
+* @param[in] iHdmiPort                 - HDMI input port.  Please refer ::dsHdmiInPort_t
+* @param[out] maxCompatibilityVersion  - Maximum Compatibility version supported by the given port
+*
+* @return dsError_t                        - Status
+* @retval dsERR_NONE                       - Success
+* @retval dsERR_NOT_INITIALIZED            - Module is not initialised
+* @retval dsERR_INVALID_PARAM              - Parameter passed to this function is invalid
+* @retval dsERR_OPERATION_NOT_SUPPORTED    - The attempted operation is not supported; e.g: source devices
+* @retval dsERR_OPERATION_FAILED           - The attempted operation has failed
+*
+* @pre dsHdmiInInit() must be called before calling this API
+*
+* @warning  This API is Not thread safe
+*
+*/
+dsError_t dsGetHdmiVersion(dsHdmiInPort_t iHdmiPort, dsHdmiMaxCapabilityVersion_t *maxCompatibilityVersion);
```

Comment:
check these  API's available in VideoPort as these will give the HDCP versions 
dsGetHDCPProtocol()
dsGetHDCPReceiverProtocol()
dsGetHDCPCurrentProtocol()
dsGetHdmiPreference()

Reply:
This API will give the maximum Hdmi Compatibile version supported for each port.